src/muya/lib/utils/exportHtml.js

```diff
@@ -88,6 +88,10 @@ class ExportHtml {
       margin-top: 0;
       display: inline-block;
     }
+    .markdown-body pre code.fenced-code-block {
```

Comment:
Would it not be better, if the text is only wrapped when exported as PDF or printed? Meaning wrap your added code inside `@media print`.